fdbserver/include/fdbserver/EncryptionOpsUtils.h

```diff
@@ -28,7 +28,10 @@
 typedef enum { TLOG_ENCRYPTION = 0, STORAGE_SERVER_ENCRYPTION = 1, BLOB_GRANULE_ENCRYPTION = 2 } EncryptOperationType;
 
 inline bool isEncryptionOpSupported(EncryptOperationType operation_type, const ClientDBInfo& dbInfo) {
```

Comment:
Can we just remove the dbInfo param from this function then. Especially since we'll change to db config soon?

Reply:
Will do.